src/app/components/Share/Swiper/SwiperButton.tsx

```diff
@@ -0,0 +1,31 @@
+import { useSwiper } from "swiper/react";
+import { cn } from "@/app/utils/helpers";
+import { IconArrowRight } from "@/app/icons/IconArrowRight";
+
+interface SwiperButtonProps {
+  direction: "next" | "prev";
+  visibility?: "always" | "largeScreen";
+}
+
+export const SwiperButton = ({ direction, visibility }: SwiperButtonProps) => {
+  const swiper = useSwiper();
+
+  return (
+    <button
+      id={`swiper-button-${direction}`}
+      className={cn(
+        "absolute top-1/2 z-10 -translate-y-1/2 transform cursor-pointer items-center justify-center rounded-full border border-black bg-white bg-opacity-50 p-2 hover:bg-opacity-75",
```

Comment:
You need to improve the styling of the button because the arrows are not in the middle, which makes it look unattractive. Set the button size to 56px by 56px as in Figma.